src/main/res/menu/conversation_list.xml

```diff
@@ -21,6 +21,10 @@
           android:id="@+id/menu_mute_selected"
           app:showAsAction="never"/>
 
+    <item android:title="@string/mark_all_as_read"
```

Comment:
I reused "mark all as read" string, because it fits for "mark all (selected chats) as read" but we could also add a new "mark as read" string

Reply:
done